frontend/.env

```
@@ -0,0 +1 @@
+VITE_STREAM_API_KEY=rjz67kbtb2tz
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)